client/app/ip/ip/block/move/ip-ip-block-move.html

```diff
@@ -73,13 +73,12 @@
 
             <div data-ng-show="!loading.save">
                 <p class="text-center">
-                    <span data-ng-bind-html="'ip_table_manage_move_ipblock_resume' | translate:{ t0: data.ipBlock.ipBlock, t1: (('ip_service' + model.serviceName.service | translate) || model.serviceName.service) }">
+                    <span data-translate="ip_table_manage_move_ipblock_resume"
+                        data-translate-values="{ t0: data.ipBlock.ipBlock, t1: ((model.serviceName.service | translate) || model.serviceName.service) }">
```

Comment:
I'm a bit confused that `t1` is configured to be:

- `model.serviceName.service | translate`
- or `model.serviceName.service`

Does it make sense to translate `model.serviceName.service` value?